### PR TITLE
MF-810: Change texts in order workspace

### PR DESCRIPTION
**Pull request by walteronoh**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
Changed the following texts in the order workspace:
"Quantity dispensed" => "Quantity"
"Pills dispensed" => "Pills to dispense"

## Screenshots
Screenshot from 2021-09-24 20-37-58

## Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-medications-app/src/order-basket/medication-order-form.component.tsx [hunk 1]
```diff
@@ -332,11 +332,11 @@ export default function MedicationOrderForm({
           </Row>
           <Row style={{ marginTop: '1rem' }}>
             <Column md={2}>
-              <FormGroup legendText={t('quantityDispensed', 'Quantity Dispensed')}>
+              <FormGroup legendText={t('quantity', 'Quantity')}>
                 <NumberInput
                   light={isTablet}
                   id="quantityDispensed"
-                  helperText={t('pillsDispensed', 'Pills dispensed')}
+                  helperText={t('pillsToDispense', 'Pills to dispense')}
```
[the remark this answers is not in the document]
Comment (brandones) [reply packages/esm-patient-medications-app/src/order-basket/medication-order-form.component.tsx:339]: @HerbertYiga Could you stop doing this, please? We get notified that the PR exists, already. People are welcome to request reviews using the "reviewers" interface.

[the remark this answers is not in the document]
Comment (HerbertYiga) [reply packages/esm-patient-medications-app/src/order-basket/medication-order-form.component.tsx:339]: > Could you stop doing this, please? We get notified that the PR exists, already. People are welcome to request reviews using the "reviewers" interface.

@brandones  well noted

[the remark this answers is not in the document]
Comment (denniskigen) [reply packages/esm-patient-medications-app/src/order-basket/medication-order-form.component.tsx:339]: @walteronoh I expect to see a corresponding update to the translation keys located in the `en.json` file in the translations directory of the medications microfrontend as a result of the `extract-translations` task being run pre-push. Did you forget to commit it?

[the remark this answers is not in the document]
Comment (walteronoh) [reply packages/esm-patient-medications-app/src/order-basket/medication-order-form.component.tsx:339]: The two keys i.e 'quantity' and 'pillsToDispense' were already there in the en.json file.

[the remark this answers is not in the document]
Comment (denniskigen) [reply packages/esm-patient-medications-app/src/order-basket/medication-order-form.component.tsx:339]: Nice! Thanks @walteronoh.

**Pull request merged**